README.rst

```diff
@@ -1,6 +1,34 @@
 MonkeyType
 ==========
 
+
+|python-versions| |circleci-badge| |codecov-badge| |pypi-badge| |pypi-download| |readthedocs-badge|
+
+.. |python-versions| image:: https://img.shields.io/pypi/pyversions/monkeytype
+   :target: https://img.shields.io/pypi/pyversions/monkeytype
+   :alt: Supported Python Versions
+
+.. |circleci-badge| image:: https://circleci.com/gh/Instagram/monkeytype/tree/master.svg?style=shield&circle-token=xxx
+   :target: https://circleci.com/gh/Instagram/monkeytype/tree/master
+   :alt: CircleCI
+
+.. |codecov-badge| image:: http://codecov.io/gh/Instagram/monkeytype/coverage.svg?branch=master
```

Comment:
For now let's leave out this badge so the PR can be merged without needing to set up codecov first.